sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -201,6 +202,7 @@ Connection getConnection() throws Exception {
   public abstract static class Read<T> extends PTransform<PBegin, PCollection<T>> {
     @Nullable abstract DataSourceConfiguration getDataSourceConfiguration();
     @Nullable abstract String getQuery();
+    @Nullable abstract List<Object> getParameters();
```

Comment:
Yes, agree. @gdubya would you try to create and use a `StatementPreparator` interface ? Else, I can do it. Please, let me know.